README.md

```diff
@@ -2,27 +2,37 @@
 
 This is an exporter for Prometheus to get Mesos data.
 
-The configuration, `config.json` should contain all Mesos slaves you want to
-monitor.
+## Building and running
 
-Prometheus configuration to add this exporter:
+    make
+    ./mesos_exporter <flags>
 
-    job: {
-      name: "mesos-exporter"
-      scrape_interval: "5s"
+### Flags
 
-      target_group: {
-        target: "http://mesosexporter.example.com/metrics"
-      }
-    }
+Name                       | Description
+---------------------------|------------
+web.listen-address         | Address to listen on for web interface and telemetry.
+web.telemetry-path         | Path under which to expose metrics.
+exporter.discovery         | Enable auto disovery of elected master and active slaves.
+exporter.discovery.master  | URL to a Mesos master.
+exporter.local-address     | Address to connect to the local slave if not using discovery.
+exporter.interval          | Interval at which to fetch the Mesos slave metrics.
 
-Where `http://mesosexporter.example.com/metrics` is the URL to this exporter.
 
-Building the binary can be done by running `make`.
+The mesos_exporter uses the [glog](https://godoc.org/github.com/golang/glog) library for logging. With the default
```

Comment:
Just btw. so you won't be surprised: this will change to use https://github.com/prometheus/log very soon now. But keep the doc as it is for now.